maintainers/maintainer-list.nix

```diff
@@ -2700,6 +2700,11 @@
     email = "info+nix@chmist.com";
     name = "karolchmist";
   };
+  kayhide = {
+    email = "kayhide@gmail.com";
+    github = "kayhide";
```

Comment:
@kayhide there is a new requirement that your `github` username be accompanied by a `githubid`. Your `githubid` is [1730718](https://api.github.com/users/kayhide).

Reply:
Got it!